set.mm

```diff
@@ -3512,6 +3512,15 @@ intermediate steps that are essentially incomprehensible to humans (other
       ( wb a1i pm5.74i ) CABABECDFG $.
   $}
 
+  ${
+    jcn.1 $e |- ( ph -> ps ) $.
+    jcn.2 $e |- ( ph -> -. ch ) $.
+    $( Inference joining the consequents of two premises.  (Contributed by
+       Glauco Siliprandi, 11-Dec-2019.) $)
+    jcn $p |- ( ph -> -. ( ps -> ch ) ) $=
+      ( wn wi jc notnotb imbi2i sylnibr ) ABCFZFZGBCGABLDEHCMBCIJK $.
```

Comment:
This looks like the deduction associated with mth8, but mth8 too has a peculiar label.

So maybe ~notim and ~notimd ? With a comment like "Sufficient condition for an implication to be false." and "Sufficient condition for an implication to be false. Deduction form."

(Rk: jc and jca should also be labeled *d)

Reply:
"jc" stands for "joining the consequents", and I think ~jcn fits to the already existing theorems  ~jc  (adding a negation to ~jc), ~jca and many other theorems prefixed with jc.

~jca cannot be renamed ~jcad, because ~jcad is already existing.

Reply:
jcad is a double deduction, so could be labeled jcadd. Independently of the precise labels, which could change, I think the convention of having *d for deductions should prevail, given its predominant use in deduction-style proofs.

This can be done later and doesn't block this PR.

Reply:
I think the revision of the jc* theorems is a separate topic and should be done in another PR.

Reply:
> I think the revision of the jc* theorems is a separate topic and should be done in another PR.

Relevant: while I mainly borrowed the metamath naming conventions for MM0 the intro/elim rules for and/or is one case where I diverged:

* and intro: MM: `jca`, MM0: `ian`
* and elim: MM: `simpl` / `simpr`, MM0: `anl` / `anr`
* or intro: MM: `olc` / `orc`, MM0: `orl` / `orr`
* or elim: MM: `jao`, MM0: `eor`

These also contain i/d versions as appropriate.

Reply:
It's nice to have mnemonics based on "and intro", "or elim", etc., which are the most standard names for these theorems/axioms. I would favor adopting this labeling scheme.